src/views/CampusSafety/views/LostAndFound/views/MissingItemEdit/index.tsx

```diff
@@ -127,7 +156,37 @@ const MissingItemFormEdit = () => {
       dateLost: new Date(formData.dateLost).toISOString() || DateTime.now().toISO(),
       forGuest: false,
     };
+    const originalFields = Object.keys(formData);
+    const newFields = Object.keys(formDataCopy);
+    let newActionNote = '';
+    for (let i = 0; i < originalFields.length; i++) {
+      if (
+        // @ts-ignore
+        JSON.stringify(formDataCopy[originalFields[i]]) != JSON.stringify(formData[newFields[i]])
```

Comment:
You can remove this ts-ignore using the following line.
```
JSON.stringify(formDataCopy[originalFields[i] as keyof typeof formDataCopy]) !==
        JSON.stringify(formData[newFields[i] as keyof typeof formData])
```

Since you know the keys in originalFields[i] newFields[i] will exist on the corresponding types, ```as keyof typeof ____``` tells typescript you are accessing a valid index.

[https://stackoverflow.com/questions/57350092/string-cant-be-used-to-index-type](https://stackoverflow.com/questions/57350092/string-cant-be-used-to-index-type)

Also, notice the use of !==, not !=.  != checks for inequality with type conversion, and not true inequality, so can lead to all sorts of weird, nasty, unexpected JS bugs (JS is a beautiful language :)